SNIPS/snip-XXX.md

```diff
@@ -0,0 +1,111 @@
+---
+snip: XX
```

Comment:
Do you want to choose a specific number or the next in line?

Reply:
Next in line please 🙏

Reply:
yeah